backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisSessionManager.java

```diff
@@ -41,7 +41,20 @@ public MolgenisSession getSession(Request request) {
 
     // get the session
     MolgenisSession session = sessions.get(request.session().id());
-    logger.info("get session for user({})", session.getSessionUser());
+    if (session.getSessionUser() == null) {
+      logger.error(
+          "got user is NULL for session key ({}). Setting user=anonymous.",
+          session.getSessionUser(),
+          request.session().id());
+      // should never happen, but in previous bug user could be null
+      // therefore make anonymous so session is valid and user can log in again
+      session.getDatabase().setActiveUser("anonymous");
```

Comment:
swallows the error , why not just fail the request , as noted in the comment should not happen , could just be a bad request

Reply:
okay, will then throw an exception. At least we will find any issues sooner :-)